<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetConfBola</class>
 <widget class="QWidget" name="WidgetConfBola">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="rbCirculo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Circulo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPatron">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Patron</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbImagen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Imagen</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="leNombre">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnColor">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImagen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>imagen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>